### PR TITLE
dd template work

### PR DESCRIPTION
**Pull request by ddietterich**
I have laid in a lot of infrastructure. It is probably time to get it into master.
Work includes:
1. Main and StartupInitialization configuration
2. Application configuration setting up beans and parameters
3. OpenAPI V3 sample YAML file describing REST endpoints with common status and error response
4. Controllers for the sample endpoints
5. Service for one of the endpoints
6. Exception scheme with base classes defined
7. Sample unit tests; one simple, one via MockMvc
8. Liquibase and implementation of MigrationService

There is probably some integration work to do to align the database setup with what has already been done.

This is a serviceable starting point. The only thing missing from my original list is setting up the Swagger UI endpoint. I"m going to leave that off for now and work on moving Stairway to its new home.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -1,11 +1,19 @@
...
+}
+
 plugins {
+	id 'java'
+	id 'idea'
 	id 'org.springframework.boot' version '2.2.2.RELEASE'
 	id 'io.spring.dependency-management' version '1.0.8.RELEASE'
-	id 'java'
-	id 'com.google.cloud.tools.jib' version '1.8.0'
+	id 'com.google.cloud.tools.jib' version '1.6.1'
```
Comment (zloery) [inline build.gradle]: Out of curiosity, why the version change on jib?

File: src/main/resources/application.properties [hunk 2]
```diff
@@ -1,4 +1,8 @@
-spring.datasource.username=${DATABASE_USER}
-spring.datasource.password=${DATABASE_USER_PASSWORD}
-spring.datasource.platform=postgres
-spring.datasource.url=jdbc:postgresql://127.0.0.1:5432/terra-kernel-k8s
\ No newline at end of file
+server.contextPath=
```
Comment (zloery) [inline src/main/resources/application.properties]: Should anything be here?

**Review by zloery — APPROVED** [review]
LGTM. I'm not familiar enough with Spring to give a thorough review, though.

File: build.gradle (hunk 1 above)
Comment (ddietterich) [reply build.gradle]: No idea. Probably a cut'n'paste error bringing something else in.
I've restored it. Thanks for catching it!

File: src/main/resources/application.properties (hunk 2 above)
Comment (ddietterich) [reply src/main/resources/application.properties]: No, there is no path. I'll try removing the entry and see if it matters.

Comment (ddietterich) [reply src/main/resources/application.properties]: Turns out it is not needed.

**Pull request merged**